opennlp-tools/src/main/java/opennlp/tools/formats/Conll03NameSampleStreamFactory.java

```diff
@@ -54,11 +55,11 @@ public static void registerFactory() {
 
     // TODO: support the other languages with this CoNLL.
     LANGUAGE lang;
-    if ("eng".equals(params.getLang())) {
+    if (LanguageCode.ENGLISH.equals(params.getLang())) {
       lang = LANGUAGE.EN;
       language = params.getLang();
     }
-    else if ("deu".equals(params.getLang())) {
+    else if (LanguageCode.GERMAN.equals(params.getLang())) {
```

Comment:
Is it German or Deutsch ? Question for the Native German speakers.

Reply:
We should have Deutsch. Or we have both.